pkg/runctl/run_manager.go

```diff
@@ -81,7 +81,7 @@ type runContext struct {
 func NewRunManager(factory k8s.ClientFactory, pipelineRunsConfig *pipelineRunsConfigStruct, secretProvider secrets.SecretProvider, namespaceManager k8s.NamespaceManager) runi.Manager {
```

Comment:
In my opinion `runi` is too short, i would suggest `runFactory`. The suggestion may be against the golnag's proverb `shorter is better` but it would better explain purpose of package.

Reply:
It is not really a factory but some interfaces.

Reply:
Yes, I would rather say it is an abstract factory because factory has a concrete implementation. Additionally, `run` is the collection/group of `Manager` and `Run` interfaces but for naming purpose I think `runInterface` would be suitable.

Reply:
The convention for package names/aliases is to use all-lowercase style and be as short as possible.

Reply:
What about `runifc`?

Reply:
Done 5c1de700ab458ea0f809a5d55672c24f1f66adb7